unicode.cpp

```diff
@@ -766,9 +772,16 @@ std::vector<std::string> unicode_regex_split(const std::string & text, const std
                 bpe_offsets = unicode_regex_split_stl(text_collapsed, regex_expr_collapsed, bpe_offsets);
             } else {
                 // no unicode category used, we can use std::wregex directly
-                const std::wstring wtext       = unicode_wstring_from_utf8(text);
```

Comment:
Did you intend to remove this?

Reply:
Thanks for the reply. I didn't see that (Between the lack of sleep, heat, and killer headache) 😅.